CONTRIBUTING.md

```diff
@@ -62,7 +62,7 @@ To ensure the lock file is up-to-date with the new versions, it's recommended to
 We use [semantic release](https://github.com/semantic-release/semantic-release) to automatically do releases based on the commit message.
 Since we are using [lerna](https://github.com/lerna/lerna) we need to specify in the commit description which packages might be affected by the release.
 
-If for some reason you have to release a module manually, please run `npm run manual-release` to publish the module. The reason for this are well stated [here](https://github.com/commercetools/nodejs/issues/223)
+If for some reason you have to release a module manually, please run `npm run release:manual` to publish the module. The reason for this are well stated [here](https://github.com/commercetools/nodejs/issues/223)
```

Comment:
Couldn't this be a yarn command too?

Reply:
Yeah, that's another thing so fix up for me.

Reply:
So chatted to @wizzy25 about this and we agreed not to change that for now. This is a public repository and we should not be as prescriptive to force people using `yarn`. In whatever environment we control we can use it: e.g. on CircleCI but otherwise we felt it's a big harsh on a public project.